README.md

```diff
@@ -2,14 +2,29 @@
 
 This is the repo for PyCon India 2020's website.
 
-India’s premier conference on using and developing the Python programming language will be held in Bangalore this year.
+India’s premier conference on using and developing the Python programming language
+is going online this year.
 
 - Conference: October 2-3
 - Workshops: October 4
 - Dev Sprints: October 4-5
 
-**Venue**
-Sheraton Grand Convention Center, Whitefield, Bengaluru
+## Getting Started
+
+This site is built with [Jekyll](https://jekyllrb.com/).
+
+* Install Jekyll. Follow the first 2 steps in https://jekyllrb.com/docs/.
+* Run `git clone https://github.com/pythonindia/inpycon2020`.
```

Comment:
I think telling the contributors to clone this repo instead of their fork might not be a good idea.

Reply:
```suggestion
* Fork the repository and clone to your machine by using `git clone https://github.com/<username>/inpycon2020`.
```